examples/realworld/ext/user/actions.js

```diff
@@ -1,20 +1,32 @@
 import HttpError from '@wasp/core/HttpError.js'
+import AuthError from '@wasp/core/AuthError.js'
+import { isPrismaError, prismaErrorToHttpError } from '@wasp/utils.js'
```

Comment:
What is tricky is that these are not part of documented Wasp JS API, right?

prismaErrorToHttpError -> that I think we want to make part of public Wasp JS API. Maybe also isPrismaError?

Reply:
I added them to the docs below @Martinsos, please let me know if that is what you had in mind. Thanks!

Reply:
Yup that is it! Just left some comments on how to possibly split those a bit.